test/runtests.sh

```diff
@@ -27,4 +26,6 @@ mocha                             \
   .test/expiretask_test.js        \
   .test/expirequeues_test.js      \
   .test/dependency_test.js        \
+  .test/workerinfo_test.js        \
```

Comment:
Oh, good catch -- most services have been updated to just use `.test/*_test.js` but I guess this one got missed..